### ISSUE TITLE
SQLEXECEX - SQLEXECEX_DEST_VARIABLE

### ISSUE THREAD

**Issue by GeorgiouJohn** [ChristianEhlscheid/vfp2c32#28]
Hi Christian
I'm John from Greece AGAIN! :grin:

I found another problem in SQLEXECEX,
When nFlags = SQLEXECEX_DEST_VARIABLE and the return field is Memo then the variable is NULL.
The older versions (<=[2.0.0.19]) of FLL work fine.
Here an example in Postgresql

```
CLEAR 
LOCAL lcFlds
lcFlds = ''
hb = SQLSTRINGCONNECT(...)
SQLEXECEX(hb,'select data_type::text from information_schema.columns limit 1','lcFlds','',2)
*SQLEXECEX(hb,'select data_type::varchar(254) from information_schema.columns limit 1','lcFlds','',2)
?lcFlds && prints .NULL.

```
Thanks for your time.

**Comment by ChristianEhlscheid** [comment]
Hello John,

thank you for the bug report, this issue is fixed in version 2.0.0.37.

Christian

John Chapter 1,23

**Comment by GeorgiouJohn** [comment]
Thanks again!
